app/page.jsx

```diff
@@ -78,16 +116,17 @@ const Dashboard = () => {
                     .sort((a, b) => b.priority - a.priority)
                     .map((filteredTicket, ticketIndex) => (
                       <TicketCard
-                        id={ticketIndex}
-                        key={ticketIndex}
+                        id={filteredTicket._id || ticketIndex}
```

Comment:
i am not able to figure out any edge case where `ticketIndex` might fail

if there are none, please change it to `ticketIndex` only